code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -16,6 +17,12 @@
 	var/mob/living/carbon/human/human_user = user
 	QDEL_IN(human_user.give_emote_overlay(/datum/bodypart_overlay/simple/emote/cry), 12.8 SECONDS)
 
+/datum/emote/living/carbon/cry/get_sound(mob/living/user)
+	var/mob/living/carbon/human_user = user
+	if(human_user.gender == FEMALE)
```

Comment:
https://github.com/tgstation/tgstation/pull/78632

Reply:
got it